### ISSUE TITLE
2.9 async changes: Accounts getNewToken meteor method implementation missing async

### ISSUE THREAD

**Issue by perbergland** [meteor/meteor#12411]
![image](https://user-images.githubusercontent.com/2690254/209917089-a940e2c8-0a2e-4aff-aa13-c5ca2d70f313.png)

The method uses await but is not marked as async.
This error is ONLY present in the release/2.9 branch, e.g. NOT in the release/2.9.1 branch which does not seem to be up-to-date nor in devel or master

```
methods.getNewToken = function () {
      const user = accounts.users.findOne(this.userId, {
        fields: { "services.resume.loginTokens": 1 }
      });
      if (! this.userId || ! user) {
        throw new Meteor.Error("You are not logged in.");
      }
      // Be careful not to generate a new token that has a later
      // expiration than the curren token. Otherwise, a bad guy with a
      // stolen token could use this method to stop his stolen token from
      // ever expiring.
      const currentHashedToken = accounts._getLoginToken(this.connection.id);
      const currentStampedToken = user.services.resume.loginTokens.find(
        stampedToken => stampedToken.hashedToken === currentHashedToken
      );
      if (! currentStampedToken) { // safety belt: this should never happen
        throw new Meteor.Error("Invalid login token");
      }
      const newStampedToken = accounts._generateStampedLoginToken();
      newStampedToken.when = currentStampedToken.when;
      await accounts._insertLoginToken(this.userId, newStampedToken);
      return accounts._loginUser(this, this.userId, newStampedToken);
    };
```

**Comment by perbergland** [comment]
Colour me confused. This change is in the PR referred to from the changelog (https://github.com/meteor/meteor/pull/12156) and in the release-2.9 branch but NOT in the 2.9 or 2.9.1 release tags.

https://github.com/meteor/meteor/blob/release-2.9/packages/accounts-base/accounts_server.js

**Comment by perbergland** [comment]
Maybe the release-2.9 branch is just incorrect - the release tags, devel and release-2.9.1 seem OK

**Comment by Grubba27** [comment]
That is really weird. Thanks for opening this issue; I will check on this one. Maybe this will be solved for now in 2.9.1?

**Comment by Grubba27** [comment]
I will close this one as It was really wrong in 2.9 but was later solved
